### ISSUE TITLE
flipbook_texture.json missing s

### ISSUE THREAD

**Issue by DBZNes** [MicrosoftDocs/minecraft-creator#912]
[Enter feedback here]
In the Custom Die Block - Animated section, 
the line "Navigate to resource pack > textures, add a flipbook_texture.json file, and insert this code:"
flipbook_texture.json missing a "s" for textures.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: b2d60b8c-f116-1f97-2d46-9d2c5a7cf4c3
* Version Independent ID: b2d60b8c-f116-1f97-2d46-9d2c5a7cf4c3
* Content: [Create an Animated Block Texture](https://learn.microsoft.com/en-us/minecraft/creator/documents/createanimatedblocktexture?view=minecraft-bedrock-stable)
* Content Source: [creator/Documents/CreateAnimatedBlockTexture.md](https://github.com/MicrosoftDocs/minecraft-creator/blob/main/creator/Documents/CreateAnimatedBlockTexture.md)
* Service: **minecraft-bedrock-edition**
* GitHub Login: @iconicNurdle
* Microsoft Alias: **mikeam**

**Comment by iconicNurdle** [comment]
Good catch! The changes will be visible after the next document release. Thank you so much!
